src/views/index.ecr

```diff
@@ -181,9 +181,10 @@
             <tr><td><code>[^abc]</code></td><td>any character except a, b, or c</td></tr>
             <tr><td><code>[a-z]</code></td><td>any character from range a-z</td></tr>
             <tr><td><code>[a-z0-9]</code></td><td>any character from range a-z or 0-9</td></tr>
+            <tr><td><code>[a|b]</code></td><td>a or b</td></tr>
```

Comment:
```suggestion
```
can we remove this line, because `[a|b]` does mean `a` or `b`, it means that any character from this set, so it's either `a` or `b` or `|` (pipe is treated as a normal character here)

Reply:
Oh didn't notice `|` is treated as a character too. Will remove it.